src/modules/tokens.ts

```diff
@@ -24,12 +25,25 @@ export namespace tokens {
               : DEFAULT_DECIMALS
           token.name = !tokenName.reverted ? tokenName.value : ""
           token.symbol = !tokenSymbol.reverted ? tokenSymbol.value : ""
-          token.totalVolume = BigInt.fromI32(0)
-
+          token.priceUSD = getPrice(tokenAddress).get("usd")
+          token.priceETH = getPrice(tokenAddress).get("eth")
```

Comment:
Same here, can you call it once than get from the result?

Reply:
solved!